src/utils/prim-lexing.fs

```diff
@@ -6,6 +6,7 @@ namespace FSharp.Compiler.Text
 
 open System
 open System.IO
+open System.Buffers
```

Comment:
Open needed here?

Reply:
Yup, not needed. I think I was trying to use ArrayPools in the lexbuffer but wanted to make a different specifically for that :) - this was left over